tests/pytest/common.py

```diff
@@ -14,3 +17,23 @@ def handler(signum, frame):
     finally:
         signal.setitimer(signal.ITIMER_REAL, 0)
         signal.signal(signal.SIGALRM, signal.SIG_DFL)
+
+def envMem(env):
+    pid = env.envRunner.masterProcess.pid
+    meminfo = psutil.Process(pid).memory_info()
+    vms = meminfo.vms / 1024
+    rss = meminfo.rss / 1024
+    return {'vsz': vms, 'rss': rss }
+
+def checkEnvMem(env, expected_vsz=None, vsz0=0, threshold=0.1, title=None):
+    if MEMINFO:
+        if title is not None:
+            print(f"--- {title}")
+        pid = env.envRunner.masterProcess.pid
+        print(paella.sh(f'cat /proc/{pid}/status | grep ^Vm', join=False))
```

Comment:
Do we always want to print?  Maybe use `env.debugPrint`?

Reply:
This runs only with `MEMINFO=1`.

Reply:
Agree with @oshadmi

Reply:
Well, I tried `env.debugPrint()` (with `RLTest --debug-print`) and got such a load of irrelevant garbage that I immediately reverted to `print()` (with `MEMINFO=1`). This is what happens when Truth (c) collides with user experience.

Reply:
Still its better to us `debugPrint`, debug prints should be verbose, you only enable it when you want better logging for debug. Also if you use `debugPrint` you will get those prints when you run with `-vv`.

Reply:
ok.